.github/workflows/manual-check-external-pr.yml

```diff
@@ -0,0 +1,44 @@
+on:
+  workflow_dispatch:
+    inputs:
+      pr-id:
+        description: 'Pull Request Id'
+        required: true
+        default: '0'
+
+jobs:
+  check-external-pr:
+    runs-on: ubuntu-latest
+    name: Create an internal branch for check the MR into the CI
+    env:
+      BOT_USER: "meta-webkit bot"
+      BOT_EMAIL: "meta-webkit@igalia.com"
+      REPO: git@github.com:Igalia/meta-webkit.git
+    steps:
+    - run: |
+       # Set the $SSH_PRIVATE_KEY environment variable in the CI to the base64
+       # encoded private key generated like this:
+       #   ssh-keygen  -t ecdsa -f runner_ecdsa
+       #   cat runner_ecdsa | base64 -w0
+       echo "Creating branch for Pull request \#${{ github.event.inputs.pr-id }}"
+       eval $(ssh-agent -s)
+       (echo "${{ secrets.SSH_PRIVATE_KEY }}" | base64 --decode | ssh-add -) || true
+       mkdir -p ~/.ssh
+       chmod 700 ~/.ssh
+       ssh-keyscan -t rsa gitlab.com github.com >>~/.ssh/known_hosts
+       git config --global user.name "${{ env.BOT_USER }}"
+       git config --global user.email "${{ env.BOT_EMAIL }}"
+       git config --global pull.rebase "preserve"
+       git clone ${{ env.REPO }} repo
+       cd repo
+       git fetch ${{ env.REPO }} pull/${{ github.event.inputs.pr-id }}/head:check/pr_${{ github.event.inputs.pr-id }}_build
+       git checkout check/pr_${{ github.event.inputs.pr-id }}_build
+       git push --force origin check/pr_${{ github.event.inputs.pr-id }}_build
```

Comment:
Why is this needed? Is the gitlab thing checking this ref? Maybe instead it can check the existing ref that corresponds to the PR?
Also, when is this ref cleaned up?

Reply:
Added  a manual action to clean up these refs in the mirror repository. We don't have a huge amount of PR activity and still to keep around the PR references in the mirror repo for a while it can be still convenient  so the clean up process can be triggered manually every then and now. 
Change: 7f3b70c2947386f64986603c7ce16c974045304b

Reply:
> Why is this needed? Is the gitlab thing checking this ref? Maybe instead it can check the existing ref that corresponds to the PR?

Gitlab Mirror is not pulling the _special_ `/pull/*` branches from Github. That is inhibiting the `push` notifications for the `pull_requests` for the Gitlab CI webhooks integration. 

This solution is based on the original idea published [here](https://gitlab.com/gitlab-org/gitlab/-/issues/5667#note_144850858) but with slightly differences:

* The logic is encapsulated in a Github action not in a `special` (check_pr) branch in the mirror
* The trigger is the **Start** button for this **Github action**

More literature here:  https://gitlab.com/gitlab-org/gitlab/-/issues/5667